mage_ai/server/app.py

```diff
@@ -93,7 +94,7 @@ def process():
     if not id:
         return
 
-    feature_set = FeatureSet(id=id)
+    feature_set = FeatureSet(id=id, api_key=api_key)
```

Comment:
Do we use the process endpoint anywhere?

Reply:
i'm not sure, but i can remove it in a future PR if we don't use this endpoint